livre/eq/Window.cpp

```diff
@@ -69,10 +69,12 @@ struct Window::Impl
                         pipe->getFrameData()->getVRParameters().getMaxGPUCacheMemoryMB();
         _textureCache.reset( new TextureCache( node->getTextureDataCache(),
                                                maxGpuMemory * LB_1MB, GL_LUMINANCE8 ));
-
-        const size_t computeThreads = 4;
+        const size_t renderThreads = 2;
+        const size_t computeThreads = 2;
         const size_t uploadThreads = 4;
         _renderPipeline.reset( new RenderPipeline( *_textureCache,
+                                                   node->getHistogramCache(),
```

Comment:
You have a per node histogram cache  but RenderPipeline is per window. This means that in single node multipipe configurations you can have multiple threads accessing the histogram cache. Is the cache thread-safe?